src/excel/utilities.ts

```diff
@@ -1,31 +1,30 @@
 /* Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. */
 
-import { CustomError } from '../errors/custom.error';
+import { APIError } from '../errors/custom.error';
 
 /**
- * Custom error type to handle API specific errors.
- */
-export class APIError extends CustomError {
-    /**
-     * @constructor
-     *
-     * @param message Error message to be propagated.
-     * @param state OAuth state if available.
-    */
-    constructor(message: string, public innerError?: Error) {
-        super('APIError', message, innerError);
-    }
-}
-
-/**
- * Helper exposing useful Utilities for Excel-Addins.
+ * Helper exposing useful Utilities for Excel Add-ins.
  */
 export class ExcelUtilities {
     /**
-     * Utility to delete a worksheet if it already exists.
-     * @returns true if the sheet had existed before being deleted.
+     * Utility to create (or re-create) a worksheet, even if it already exists.
+     * @returns the new worksheet
      */
-    static async forceCreateSheet(workbook: Excel.Workbook, sheetName: string): Promise<Excel.Worksheet> {
+    static async forceCreateSheet(
+        workbook: Excel.Workbook,
+        sheetName: string,
+        options?: {
```

Comment:
Well I don't think that is necessary. The function documentation and API call will explain what that boolean stands for. Its really annoying to pass an options bag. I'll be updating it while merging.